ci/cases/yamls/soca_gfs_defaults_ci.yaml

```diff
@@ -0,0 +1,5 @@
+defaults:
+  !INC {{ HOMEgfs }}/parm/config/gfs/yaml/defaults.yaml
+base:
+  ACCOUNT: {{ 'SLURM_ACCOUNT' | getenv }}
+  DO_JEDIOCNVAR: "NO"
```

Comment:
Is that `DO_JEDIOCNVAR` switch flipped back on somewhere else? If not then we probably didn't run `soca` in the test.

Reply:
oops